docs/release-notes/next/PT-162429019-disable-internal-debug-api-by-default.md

```diff
@@ -0,0 +1,2 @@
+* Disable internal debug API endpoints by default. To enable setup epoch.yaml `http > internal > debug_endpoints` to `true`
+* Mark `http > endpoints > debug` and `http > debug` configuration params as deprecated
```

Comment:
@fabiankrol Just ensuring we are on the same page - removal of these is planned for Minerva, right? Cc @dincho

Reply:
[Tracked](https://www.pivotaltracker.com/story/show/163387507).

Reply:
Correct 👌